jira/resources.py

```diff
@@ -138,8 +137,8 @@ class Resource:
         "closed",
     )
 
-    # A list of properties that should uniquely identify a Resource object
-    # Each of these properties should be hashable, usually strings
+    # A list of properties that should uniquely identify a Resource object Each of
+    # these properties should be hashable, usually strings
```

Comment:
Why do we add the "Each of" to a new line here? if it made sense to do so we should continue the right case for a sentence, i.e. "each of"